### DISCUSSION TITLE
请求恢复PCL2窗口最大化按钮

### DISCUSSION THREAD

**Discussion by Mbb-minecraft** [Meloong-Git/PCL#4408] [讨论]
描述：像大多数窗口一样，恢复（在Dev 0.0.2中被支持[[Dev 0.0.2 | 最大化与滑动条](https://afdian.com/p/c6b18d849b9a11e8bcc552540025c377)]；Dev 0.0.11中被删除[[Dev 0.0.11 | 重绘与捉虫](https://afdian.com/p/d2b36e90a2ca11e89c7452540025c377)]）最大化的按钮。
PS：很不理解龙腾猫跃为什么把这个按钮删掉了......

**Comment by allMagicNB** [comment]
那么，为什么要最大化？

**Comment by Deep-Dark-Forest** [comment]
#90
